sql/branch.sql

```diff
@@ -0,0 +1,3 @@
+BEGIN;
+    UPDATE payment_instructions SET due = 0 WHERE due > 9.41;
```

Comment:
Hmm, maybe we could use https://github.com/gratipay/gratipay.com/issues/3851 to truly solve this?